account_renumber/test/create_moves.py

```diff
@@ -94,6 +94,7 @@ def create_lots_of_account_moves(dbname, user, passwd, howmany):
 # ------------------------------------------------------------------------
 # ------------------------------------------------------------------------
 
+
```

Comment:
this change should not behere

Reply:
If I remove this change it is breaking the build violating pep E305 convention.